### ISSUE TITLE
result for verification

### ISSUE THREAD

**Issue by DoubleKing0625** [MahdiHajibabaei/unified-embedding#4]
Hi，

I tried to reproduce your best verification result. But I always got little gap with your best result in paper, like 4.89 compared with 4.30 on ERR(%) and  0.419 compared with 0.413 on C_min_det.
And may you share the log for training best verification model?

Thanks in advance.

**Comment by MahdiHajibabaei** [comment]
Hi,

I will train the AMS-20_modified.prototxt, again and will keep you updated but in the meantime, make sure that in the AMS-20_modified.prototxt:
1. The embedding dimensionality or num_output: in "fc5" is set to 128.
2. Dropout is applied:
layer {
  name: "drop6"
  type: "Dropout"
  bottom: "res4_3p"
  top: "res4_3p"
  dropout_param {
    dropout_ratio: 0.5
  }
}
3. Scale parameter is at least 50:
layer {
  name: "fc6_scale"
  type: "Scale"
  bottom: "fc6"
  top: "fc6_scale"
  param {
    lr_mult: 1
    decay_mult: 0
  }
  scale_param {
    filler {
      value: 50
    }
    bias_term: true
  }
}
4. The label specific add is at most -20 (-25 and etc. would also work):
layer {
  name: "label_scale_margin"
  type: "LabelSpecificAdd"
  bottom: "fc6_scale"
  bottom: "label"
  top: "fc6_scale_margin"
  label_specific_add_param {
    bias: -20
  }
}

Best,
Mahdi Hajibabaei

**Comment by DoubleKing0625** [comment]
Ok. I will try this hyper-parameter setting and wait for your update.
Thanks~

**Comment by MahdiHajibabaei** [comment]
Hi,

Also note that evaluating the models on few pairs chosen by collectors of VoxCeleb is not a statistically sound evaluation and it is much better to compare the EER and DCF on all pairs by using "roc.py".

**Comment by MahdiHajibabaei** [comment]
Hi,

I wanted to know whether you got the desired EER so that I can close this issue.

Best,
Mahdi Hajibabaei

**Comment by DoubleKing0625** [comment]
Finally I got ERR(%) 4.31, but it's ok.
Thanks.
